src/Tests/Microsoft.NET.Publish.Tests/GivenThatWeWantToPublishAProjectWithDependencies.cs

```diff
@@ -37,19 +38,18 @@ public void It_publishes_projects_with_simple_dependencies()
                 .Should()
                 .Pass();
 
-            DirectoryInfo publishDirectory = publishCommand.GetOutputDirectory();
+            DirectoryInfo publishDirectory = publishCommand.GetOutputDirectory(targetFramework);
 
             publishDirectory.Should().OnlyHaveFiles(new[] {
                 "SimpleDependencies.dll",
                 "SimpleDependencies.pdb",
                 "SimpleDependencies.deps.json",
                 "SimpleDependencies.runtimeconfig.json",
                 "Newtonsoft.Json.dll",
-                "System.Runtime.Serialization.Primitives.dll",
-                "System.Collections.NonGeneric.dll",
+                "SimpleDependencies.exe"
```

Comment:
On Mac and Linux, the executable doesn't have an `.exe` extension, and we have the `EnvironmentInfo.ExecutableExtension` to help with that.

```suggestion
                $"SimpleDependencies{EnvironmentInfo.ExecutableExtension}"
```